cfme/tests/intelligence/test_chargeback_assignments.py

```diff
@@ -17,13 +15,8 @@
 ]
```

Comment:
In that case, I'd kinda rather have it there (and be redundant) then try to trick linter.